src/pyastar2d/astar_wrapper.py

```diff
@@ -19,14 +19,28 @@
     ctypes.c_int,   # start index in flattened grid
     ctypes.c_int,   # goal index in flattened grid
     ctypes.c_bool,  # allow diagonal
+    ctypes.c_int,   # heuristic_override
+    ctypes.c_int,   # tiebreaker_coefficient
 ]
 
 
 def astar_path(
         weights: np.ndarray,
         start: Tuple[int, int],
         goal: Tuple[int, int],
-        allow_diagonal: bool = False) -> Optional[np.ndarray]:
+        allow_diagonal: bool = False,
+        heuristic_override: int = 0,
+        tiebreaker_coefficient: int = 0) -> Optional[np.ndarray]:
```

Comment:
I'm wondering if we should instead define a new function for uniform grids with these tiebreaking arguments to make it clear that the solution is no longer strictly a shortest path depending on the choice of arguments.

Reply:
By default it is shortest path, if users are overriding the heuristic or tiebreaker they better know what they are doing

Maybe we can just add a comment in the docstring? That shortest path guarantee and maybe others are out of the door when using the too new parameters?

Reply:
Added below to docstring:

param float tiebreaker_coefficient: Add tiebreaker to heuristic cost, 0=disable, positive enables it by that amount/1000.0
    
    Important: Please take care when using heuristic_override and tiebreaker_coefficient, they may not take the shortest path.